api/main.go

```diff
@@ -30,16 +31,26 @@ func main() {
 	log.Println("starting sever")
 	db := database_init()
 
+	r := gin.Default()
+
 	// Initialize roles
 	InitializeRoles(db)
 
-	r := gin.Default()
-
-	r.GET("/roles", GetRolesHandler(db))
+	r.GET("/roles", Handlers.GetRolesHandler(db))
+	r.GET("/roles/:id", Handlers.GetRoleByIdHandler(db))
 
-	r.POST("/user", CreateUserHandler(db))
+	r.POST("/user", Handlers.CreateUserHandler(db))
```

Comment:
We can move create user to a separate /auth/register route when implementing the auth.